ccp.h

```diff
@@ -208,6 +208,11 @@ void ccp_connection_free(u16 sid);
  */
 struct ccp_connection *ccp_connection_lookup(u16 sid);
 
+
+/* Lookup a datapath program, available to all flows
+ */
+struct DatapathProgram* datapath_program_lookup(u16 pid);
```

Comment:
Not that `ccp.h` is particularly clean or whatever, but does this need to be part of the public libccp API headerfile?